test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpAnalyzerVerifier`1.cs

```diff
@@ -71,9 +71,9 @@ public static DiagnosticResult Diagnostic (DiagnosticDescriptor descriptor)
 		}
 
 		/// <inheritdoc cref="AnalyzerVerifier{TAnalyzer, TTest, TVerifier}.VerifyAnalyzerAsync(string, DiagnosticResult[])"/>
-		public static async Task VerifyAnalyzerAsync (string src, params DiagnosticResult[] expected)
+		public static async Task VerifyAnalyzerAsync (string src, (string, string)[]? analyzerOptions = null, params DiagnosticResult[] expected)
```

Comment:
To avoid having to add `expected:` everywhere, you could add an overload instead of an optional parameter